Azure/sonic-buildimage
# [libteam] Upgrade libteam to 1.28-1 and reorganize patches

Signed-off-by: yorke <yorke.yuan@asterfusion.com>



**- What I did**
Upgrade libteam to 1.28-1 and reorganize patches.

**- How I did it**
Modify Makefile and patch serires.
1. Advance libteam to jpirko/libteam 5c5e498
2. Advance debian supporting to debian/libteam.git 7188c361
3. Remove patch 0001, 0010 and 0011 as they are already including in the new source code
4. Reorganize the other patches simultaneously
Note: NETLINK_RCVBUF is 196608 now (prev patch set it to 983040) and it works fine for testing. (Refering to libteam: double NETLINK_RCVBUF to fix -ENOMEM error)

**- How to verify it**
Build a SONiC image and run the lacp test in the sonic-mgmt repo. Ensure the test succeeds.

**- Description for the changelog**
	modified:   rules/libteam.mk
	modified:   src/libteam/Makefile
	new file:   src/libteam/patch/series
	deleted:    src/libteam/series
	deleted:    src/libteam/0001-libteam-Add-team_get_port_enabled-function.patch
	deleted:    src/libteam/0010-teamd-lacp-update-port-state-according-to-partners-sy.patch
	deleted:    src/libteam/0011-libteam-resynchronize-ifinfo-after-lost-RTNLGRP_LINK-.patch
	renamed:    src/libteam/0002-libteam-Temporarily-remove-redundant-debug-mes.patch -> src/libteam/patch/0001-Temporarily-remove-redundant-debug-messages.patch
	renamed:    src/libteam/0003-teamd-lacp-runner-will-send-lacp-update-right-after-.patch -> src/libteam/patch/0002-Lacp-runner-will-send-lacp-update-right-after-it-rec.patch
	renamed:    src/libteam/0004-libteam-Add-lacp-fallback-support-for-single-member-.patch -> src/libteam/patch/0003-Add-fallback-support-for-single-member-port-LAG.patch
	renamed:    src/libteam/0005-libteam-Add-warm_reboot-mode.patch -> src/libteam/patch/0004-Add-warm_reboot-mode.patch
	renamed:    src/libteam/0007-Skip-setting-the-same-hwaddr-to-lag-port-to-avoid-di.patch -> src/libteam/patch/0005-Skip-setting-the-same-hwaddr-to-lag-port-to-avoid-di.patch
	renamed:    src/libteam/0012-teamd-do-not-process-lacpdu-before-the-port-ifinfo-i.patch -> src/libteam/patch/0006-teamd-do-not-process-lacpdu-before-the-port-ifinfo-i.patch
	renamed:    src/libteam/0013-teamd-lacp-port-admin-down-recv-not-processing.patch -> src/libteam/patch/0007-Administratively-shutdown-port-channel-has-member-po.patch




**- A picture of a cute animal (not mandatory but encouraged)**
